### PR TITLE
migrate the java docker image from _/java to _/openjdk

### PR DESCRIPTION
**Pull request by PierreBesson**
Fix https://github.com/jhipster/generator-jhipster/issues/4127

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM java:openjdk-8-jdk-alpine
+FROM openjdk:8-jre-alpine
```
Comment (pascalgrimaud) [inline Dockerfile]: must be: **jdk** instead of **jre**

File: src/main/docker/dev/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM java:openjdk-8-jdk-alpine
+FROM openjdk:8-jre-alpine
```
Comment (pascalgrimaud) [inline src/main/docker/dev/Dockerfile]: as the war is built locally and copy during the docker build, **jre** is enough here :)

Comment (PierreBesson) [inline src/main/docker/dev/Dockerfile]: Thanks ! I almost broke everything :smile:

**Comment by PierreBesson** [comment]
@jdubois please merge this it's been a long time 😄.

**Comment by jdubois** [comment]
Sorry I missed it... Yes I'm merging!

**Pull request merged**